views/index.dt

```diff
@@ -0,0 +1,21 @@
+!!! 5
```

Comment:
That's short for `doctype html`, got used to that, but indeed it's not mentionend at [Jade - doctype](http://jade-lang.com/reference/doctype/).